wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -30,6 +38,197 @@ sealed abstract class Wallet extends LockedWallet with UnlockedWalletApi {
     LockedWallet(nodeApi, chainQueryApi)
   }
 
+  /** We need a output so we can get proper tx size estimation */
+  private def getDummyDLCFundingOutputs(
+      amount: CurrencyUnit): Vector[TransactionOutput] = {
+    val key = ECPublicKey.freshPublicKey
+    val spk = P2WSHWitnessSPKV0(MultiSignatureScriptPubKey(2, Seq(key, key)))
+
+    // funded output and the other party's change
+    Vector(TransactionOutput(amount, spk), TransactionOutput(Satoshis.one, spk))
+  }
+
+  /** TODO Not currently used, will need to update wallet implementation for support  */
+  def writeDLCAddresses(eventId: String): Future[Vector[AddressDb]] = {
```

Comment:
nit: Use a hash digest instead of a string

Reply:
also some documentation around all the magic happening here

Reply:
This will likely just be replaced by #1111